Updated Buidler to Hardhat (rebranded) (#1819)
ethereum/ethereum-org-website: src/content/developers/docs/development-networks/index.md

- [GitHub](https://github.com/trufflesuite/ganache)
- [Documentation](https://www.trufflesuite.com/docs/ganache/overview)

### Hardhat EVM

A local Ethereum network designed for development. It allows you to deploy your contracts, run your tests and debug your code

> Hardhat EVM come built-in with Hardhat, a task runner for Ethereum smart contract developers.

Reviewer: ```suggestion
Hardhat EVM come built-in with Hardhat, an Ethereum development environment for professionals.
```

> Hardhat EVM come built-in with Hardhat, an Ethereum development environment for professionals.

Reviewer 2: ```suggestion
Hardhat Network comes built-in with Hardhat, an Ethereum development environment for professionals.
```
